tests/helpers.js

```diff
@@ -134,7 +134,8 @@ export function makeSureItFails(): Function {
  * assert.equal(fakeProcess.exit.called, true);
  *
  */
-export function fake(original: Object, methods: Object = {}): Object {
+ // $FLOW_IGNORE: fake can return any kind of object and fake a defined set of methods for testing.
+export function fake<T>(original: Object, methods: Object = {}): T {
```

Comment:
does `$FLOW_IGNORE` cause it to completely ignore this line? Will these annotations be completely ignored? In other words, does the `<T>` polymorphic arg have any affect?